tools/wid_usage.py

```diff
@@ -0,0 +1,7 @@
+
+from autopts.utils import extract_wid_testcases_to_csv
+
+# USE: Simply  run python3 -m tools.wid_usage in auto-pts directory
```

Comment:
I suppose this assumes that some tests have already been run and the logs are in the correct location relative to the CWD?

Reply:
Was wondering if this (simple) tool should perhaps have an argparser as well with a path to the logs (defaulting to e.g. ./logs)? 

But that can also wait until there's an actual request for that feature